pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarConfig.java

```diff
@@ -52,45 +56,50 @@ public class PulsarConfig {
   private final SubscriptionInitialPosition _subscriptionInitialPosition;
   private final String _authenticationToken;
   private final String _tlsTrustCertsFilePath;
+
+  private final String _issuerUrl;
+  private final String _credentialsFilePath; // Absolute path of your downloaded key file
```

Comment:
How will someone provide this file in pinot deployments? I think if it's running in docker, expecting folks to copy the credentials file to docker volumn is wrong choice.

Reply:
Is there a way we can mention encrypted version of this in table config or mention something like S3 path?

Reply:
Ok,in that case mention a small TODO on top of it that mentions to find a good way to push this file to all servers